test/basic.spec.js

```diff
@@ -59,4 +59,14 @@ describe("Basic Tests", function() {
 
     assert.equal(template(context), "&lt;html&gt; tag with &#39;single&#39; &amp; &quot;double&quot; quotes");
   });
+	
+  it("should not escape certain characters escape has been disabled", function() {
+    var template = templater("{{foo}}"),
+        context = {
+          foo: "<html> tag with 'single' & \"double\" quotes",
+        };
+
+    assert.equal(template(context, false), "<html> tag with 'single' & \"double\" quotes");
```

Comment:
```suggestion
    assert.equal(template(context, { autoescape: false }), "<html> tag with 'single' & \"double\" quotes");
```

Again, open to a different name than `autoescape`